assets/scss/components/_avatar.scss

```diff
@@ -122,8 +143,15 @@ Class                     | Description
 .avatar--noPhoto {
 	position: relative;
 	text-indent: initial;
-	opacity: 0.54; // match text--secondary alpha
+	background-color: darken($avatarBgAsHex, ((1 - $noPhotoIconOpacity - alpha($avatarBg))*100)*1%);
 	background-position-y: 4px !important;
+	&:before {
+		background-color: transparentize($C_contentBG, 1-$noPhotoIconOpacity);
```

Comment:
I'd recommend using `$C_white` instead. The content BG is a separate concept and may be subject to change (and in this case, we always want a white scrim to lighten the avatar)